jp/components/splitter.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
Don't forget to copy paste the latest en version so that the translation doesn't differ.